gitpod-io/website
# Re-enable example workspace option from CTA

## Description
This partially reverts the changes from #1140 re-enabling the link from `Try Now` and `Sign Up` to the `#get-started` section which allows users to select example workspaces.

preview: https://deploy-preview-1195--gitpod-kumquat.netlify.app/

## Related Issue(s)
Fixes #1194 

## How to test
Clicking on Try Now or Sign Up should not open the dashboard, but scroll to Get Started on the home page.

## Release Notes
```release-note
The "Try Now" and "Sign Up" buttons on the website homepage link to the Get Started section of the homepage.
```

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/1195"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
